lib/kapnismology/terminal.rb

```diff
@@ -1,20 +1,26 @@
 module Kapnismology
   class Terminal
-
     def self.green(msg)
-      "\e[32m\e[1m#{msg}\e[0m"
+      colorize(msg, "\e[32m")
     end
 
     def self.red(msg)
-      "\e[31m\e[1m#{msg}\e[0m"
+      colorize(msg, "\e[31m")
     end
 
     def self.yellow(msg)
-      "\e[33m\e[1m#{msg}\e[0m"
+      colorize(msg, "\e[33m")
     end
 
     def self.bold(msg)
-      "\e[1m#{msg}\e[0m"
+      colorize(msg, '')
+    end
+
+    private
+    def self.colorize(msg, color_code)
+      # \e[1m adds bold font, \e[0m resets all styles
+      # Colorize interactive terminals for humans to enjoy.
```

Comment:
[pedantic nits] sorry if I wasn't clear but I referred to only the line about the codes, not the "colorize interactive..." one, which should be the method's comment or plain removed 😅